src/Common/src/CoreLib/System/Text/ValueStringBuilder.cs

```diff
@@ -174,6 +174,25 @@ public unsafe void Append(ReadOnlySpan<char> value)
             _pos += value.Length;
         }
 
+        public unsafe void AppendReversed(ReadOnlySpan<char> value)
```

Comment:
@jkotas are you fine with adding these helper to ValueStringBuilder or should I fork them off?

Reply:
You can add this in your local copy (ValueStringBuilder.AppendReverse.cs). I do not think it should be in the common one.